recipes/metaspace2020/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "metaspace2020" %}
+{% set version = "2.0.9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: a1c4dbd8aaf8869c3266ac210a921a3065165a4ea9668465b479c242e4d8af94
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation"
+  noarch: python
+  run_exports:
+    - {{ pin_subpackage('metaspace2020', max_pin='x.x') }}
+
+requirements:
+  host:
+    - matplotlib-base
+    - numpy
+    - pandas
+    - pillow
+    - pip
+    - plotly >=1.12
+    - python
+    - requests
+    - tqdm
```

```suggestion
    - pip
    - python
```

normally this should be enough